### ISSUE TITLE
Can't find ".../sandbox/mingw-w64-i686/bin/i686-w64-mingw32-gcc"

### ISSUE THREAD

**Issue by qupfer** [rdp/ffmpeg-windows-build-helpers#75]
Hi,

on my arch system, your (normally really useful^^) script doesn't work not as usual :-(
The german phrase "Datei oder Verzeichnis nicht gefunden" means simply "(file or directory) not found"
```
[...]
done downloading dlfcn-win32
testing compiler: /home/qupfer/sandbox/mingw-w64-i686/bin/i686-w64-mingw32-gcc -shared -o /tmp/test.dll /tmp/test.c
./configure: Zeile 163: /home/qupfer/sandbox/mingw-w64-i686/bin/i686-w64-mingw32-gcc: Datei oder Verzeichnis nicht gefunden
/home/qupfer/sandbox/mingw-w64-i686/bin/i686-w64-mingw32-gcc could not create shared file with Windows API functions.
Make sure your MinGW system is working properly.

making /home/qupfer/sandbox/win32/dlfcn-win32 as $ PATH=/home/qupfer/sandbox/mingw-w64-i686/bin:/usr/lib/hardening-wrapper/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl make  -j 2
Makefile:4: config.mak: Datei oder Verzeichnis nicht gefunden
make: *** Keine Regel, um „config.mak“ zu erstellen.  Schluss.
```

The last lines of build.log are:
```
make use of stdint.h in include/isl/stdint.h (assuming C99 compatible system)
checking which gmp to use... system
checking gmp.h usability... yes
checking gmp.h presence... yes
checking for gmp.h... yes
checking for main in -lgmp... yes
configure: error: gmp library too old
```

The problem looks like, that the downloaded 5.3 is too old for some used dependencies installed on my system. I will try to change simply the download-link, but I'm not very hopeful that this will be enough ;)
If you (or anybody else) has a solution, it would be great. 

And just as a small note. 
On a fresh installed ubuntu 15.04, the script complained about a missing libtool, which is - of course - installed. Removing the "check" solve this stumbling block.

**Comment by rdp** [comment]
What OS initially? If you nuke sandbox and try again same result?

**Comment by qupfer** [comment]
Arch Linux x86_64.
And yes, "rm -rf sandbox" doesn't change the result. Same error. I will try a "virgin" arch-vm to check, if its a "arch" problem or just my server.

**Comment by rdp** [comment]
OK I "hopefully" fixed it for linux libtool, seems to be working fine here
for ubuntu, could you try it again? Unfortunately I have little [zero]
experience with Arch, maybe you can help?
-roger-

**Comment by rdp** [comment]
OK edits don't get sent out as notifications, only saw it now. Closing.

**Comment by FallingSnow** [comment]
I am getting the same error. I have installed `gmp 6.1.1` but still get the error.

**Comment by rdp** [comment]
you're getting gmp library too old? what if you just run your equivalent of

/home/qupfer/sandbox/mingw-w64-i686/bin/i686-w64-mingw32-gcc

**Comment by FallingSnow** [comment]
How would I go about doing that?

Also, I found a strange line in build.log

`../source/isl-0.14/configure: line 3419: cd: /root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/isl/isl-0.14-x86_64: No such file or directory`

Here is a larger portion of the log containing the line above.
```
----------------------------------------------------------------------
Libraries have been installed in:
   /root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/mpc-1.0.3-x86_64/lib

If you ever happen to want to link against installed libraries
in a given directory, LIBDIR, you must either use libtool, and
specify the full pathname of the library, or use the '-LLIBDIR'
flag during linking and do at least one of the following:
   - add LIBDIR to the 'LD_LIBRARY_PATH' environment variable
     during execution
   - add LIBDIR to the 'LD_RUN_PATH' environment variable
     during linking
   - use the '-Wl,-rpath -Wl,LIBDIR' linker flag
   - have your system administrator add LIBDIR to '/etc/ld.so.conf'

See any operating system documentation about shared libraries for
more information, such as the ld(1) and ld.so(8) manual pages.
----------------------------------------------------------------------
make[2]: Nothing to be done for 'install-data-am'.
make[2]: Leaving directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build/src'
make[1]: Leaving directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build/src'
Making install in tests
make[1]: Entering directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build/tests'
make[2]: Entering directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build/tests'
make[2]: Nothing to be done for 'install-exec-am'.
make[2]: Nothing to be done for 'install-data-am'.
make[2]: Leaving directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build/tests'
make[1]: Leaving directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build/tests'
Making install in doc
make[1]: Entering directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build/doc'
make[2]: Entering directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build/doc'
make[2]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/mpc-1.0.3-x86_64/share/info'
 /usr/bin/install -c -m 644 ../../source/mpc-1.0.3/doc/mpc.info '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/mpc-1.0.3-x86_64/share/info'
 install-info --info-dir='/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/mpc-1.0.3-x86_64/share/info' '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/mpc-1.0.3-x86_64/share/info/mpc.info'
make[2]: Leaving directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build/doc'
make[1]: Leaving directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build/doc'
make[1]: Entering directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build'
make[2]: Entering directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build'
make[2]: Nothing to be done for 'install-exec-am'.
 /usr/bin/mkdir -p '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/mpc-1.0.3-x86_64/include'
 /usr/bin/install -c -m 644 ../source/mpc-1.0.3/src/mpc.h '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/mpc-1.0.3-x86_64/include'
make[2]: Leaving directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build'
make[1]: Leaving directory '/root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/mpc/build'
2016-07-21 09:35:29 URL: ftp://gcc.gnu.org/pub/gcc/infrastructure/isl-0.14.tar.bz2 [1399896] -> "isl-0.14.tar.bz2" [1]
configure: WARNING: unrecognized options: --with-piplib
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /usr/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking whether make supports nested variables... (cached) yes
../source/isl-0.14/configure: line 3419: cd: /root/Documents/ffmpeg-windows-build-helpers/ffmpeg_build/sandbox/cross_compilers/pkgs/isl/isl-0.14-x86_64: No such file or directory
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking for style of include used by make... GNU
checking dependency style of gcc... gcc3
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking dependency style of g++... gcc3
checking for C compiler vendor... gnu
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking whether C compiler accepts -malign-double... yes
checking whether C compiler accepts -fstrict-aliasing... yes
checking whether C compiler accepts -ffast-math... yes
checking for gcc architecture flag... 
checking for x86 cpuid 0 output... a:756e6547:6c65746e:49656e69
checking for x86 cpuid 1 output... 10676:3040800:ce3bd:bfebfbff
checking whether C compiler accepts -march=pentium3... no
checking whether C compiler accepts -mcpu=pentium3... no
checking whether C compiler accepts -mpentium3... no
checking whether C compiler accepts -march=pentiumpro... no
checking whether C compiler accepts -mcpu=pentiumpro... no
checking whether C compiler accepts -mpentiumpro... no
checking for gcc architecture flag... unknown
checking whether C compiler accepts -O3 -fomit-frame-pointer -malign-double -fstrict-aliasing -ffast-math... yes
checking whether the compiler supports function __attribute__((__warn_unused_result__))... yes
checking for __attribute__... yes
checking how to print strings... printf
checking for a sed that does not truncate output... /usr/bin/sed
checking for grep that handles long lines and -e... /usr/bin/grep
checking for egrep... /usr/bin/grep -E
checking for fgrep... /usr/bin/grep -F
checking for ld used by gcc... /usr/lib/hardening-wrapper/bin/ld
checking if the linker (/usr/lib/hardening-wrapper/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 1572864
checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/lib/hardening-wrapper/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for mt... no
checking if : is a manifest tool... no
checking how to run the C preprocessor... gcc -E
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... yes
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/usr/lib/hardening-wrapper/bin/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... no
checking whether to build static libraries... yes
checking how to run the C++ preprocessor... g++ -E
checking for ld used by g++... /usr/lib/hardening-wrapper/bin/ld -m elf_x86_64
checking if the linker (/usr/lib/hardening-wrapper/bin/ld -m elf_x86_64) is GNU ld... yes
checking whether the g++ linker (/usr/lib/hardening-wrapper/bin/ld -m elf_x86_64) supports shared libraries... yes
checking for g++ option to produce PIC... -fPIC -DPIC
checking if g++ PIC flag -fPIC -DPIC works... yes
checking if g++ static flag -static works... yes
checking if g++ supports -c -o file.o... yes
checking if g++ supports -c -o file.o... (cached) yes
checking whether the g++ linker (/usr/lib/hardening-wrapper/bin/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... (cached) GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking for perl... perl
checking for pdflatex... no
checking for pod2html... pod2html
checking for stdint types... stdint.h (shortcircuit)
make use of stdint.h in include/isl/stdint.h (assuming C99 compatible system)
checking which gmp to use... system
checking gmp.h usability... yes
checking gmp.h presence... yes
checking for gmp.h... yes
checking for main in -lgmp... yes
configure: error: gmp library too old
```

**Comment by rdp** [comment]
hopefully fixed in the latest, see #177
